manifest.json

```diff
@@ -14,6 +14,7 @@
     "open_in_tab": true
   },
   "permissions": [
+    "downloads",
```

Comment:
Adding new permissions is dangerous, this can be circumvented by [dynamically requesting for permission](https://developer.chrome.com/apps/permissions)

Reply:
Good idea, might be difficult to implement though, but worth checking out.

Reply:
It is not difficult at all, I'm also using it in a current draft